### PR TITLE
fix(AvatarWrapper): render avatars for business profiles

### PR DESCRIPTION
**Pull request by fergusjordantw**
## 🖼 Context

Currently, we aren't rendering business profiles avatars, we always show the briefcase icon.

## 🚀 Changes

Updates the logic to show the url passed in, regardless of profile type.

## ✅ Checklist

- [x] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [x] Changes are tested and all tests pass
- [x] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [x] Changes work in all supported browsers (don't forget IE11)
- [x] You've updated the documentation if necessary

### CODE REVIEW

**Review by mikknarusk — APPROVED** [review]

File: packages/components/src/common/avatarWrapper/AvatarWrapper.js [hunk 1]
```diff
@@ -18,17 +18,12 @@ export default function AvatarWrapper({ url, profileType }) {
...
+          {isBusinessProfile ? <BriefcaseIcon /> : <ProfileIcon />}
         </Avatar>
       )}
-      {!isBusinessProfile && !renderImage && (
-        <Avatar type={Avatar.Type.ICON} size={Avatar.Size.MEDIUM}>
-          <ProfileIcon />
-        </Avatar>
-      )}
-      {!isBusinessProfile && renderImage && (
+      {renderImage && (
```
Comment (dozortsev) [inline packages/components/src/common/avatarWrapper/AvatarWrapper.js:26]: can we please just use ternary here:

```jsx
{renderImage
  ? <Avatar type={Avatar.Type.THUMBNAIL}>...
  : <Avatar type={Avatar.Type.ICON}>...
}

Comment (dozortsev) [reply packages/components/src/common/avatarWrapper/AvatarWrapper.js:26]: or `if-else` 🙂

Comment (dozortsev) [reply packages/components/src/common/avatarWrapper/AvatarWrapper.js:26]: it would be easier to the code which would help avoid similar bugs in future

Comment (fergusjordantw) [reply packages/components/src/common/avatarWrapper/AvatarWrapper.js]: sure

**Review by dozortsev — APPROVED** [review]

**Pull request merged**